shared/src/store/action-reducers/simulated-region.ts

```diff
@@ -268,4 +280,24 @@ export namespace SimulatedRegionActionReducers {
             },
             rights: 'participant',
         };
+
+    export const removeBehaviorToSimulatedRegion: ActionReducer<RemoveBehaviorFromSimulatedRegionAction> =
```

Comment:
I think this should be called `removeBehaviorFromSimulatedRegion` (with `From` instead of `To`)

Reply:
Yes I misstyped that for some reason

Reply:
done